src/OpenTelemetry/Metrics/Aggregators/DoubleCounterSumAggregator.cs

```diff
@@ -42,9 +42,9 @@ public override MetricData ToMetricData()
         {
             return new DoubleSumData
             {
-                StartTimestamp = new DateTime(this.GetLastStartTimestamp().Ticks),
+                StartTimestamp = new DateTime(this.GetLastStartTimestamp().Ticks, DateTimeKind.Utc),
```

Comment:
We could move this to the base class and remove all these changes. What do you think?

Reply:
Ok. I gave it a try and created `GetLastEndTimestampAsDateTime` and `GetLastStartTimestampAsDateTime` in the base class. Let me know if you have an opinion on better naming.

The base class methods decrease the risk of an inherited class making that mistake too. On the other hand it increases API surface. Let me know what you think works best.

Reply:
I was thinking something a little different. DoubleSumData is based on `MetricData`, which has the StartTimestamp property. What we could do is create a constructor there and in the MetricData constructor we would create the timestamp. With that, all places where we start `xyz_SumData` would be simplified. Don't know if that is possible, but at least it would prevent new Metric classes using the wrong timestamp.